src/components/Chat/BottomSheetGif.js

```diff
@@ -109,6 +121,7 @@
         refreshing={isLoading}
         onRefresh={onLoad}
         numColumns={2}
+        ListFooterComponent={<View style={{height: 100}} />}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a `ListFooterComponent` with a height of 100 may improve the user experience by providing some space below the list. However, ensure that this doesn't interfere with any other UI elements or functionality.
```diff
+ ListFooterComponent={<View style={{height: 100}} />}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->